README.md

```diff
@@ -148,9 +150,8 @@ Features:
 * Custom parameter types: ```@Request.(Header|PathParam|QueryParam)(encoder=MyCustomParamTypeEncoder.class)```.
 * Custom serialization/deserialization: add ```@Request.Body(MySerializableTypeBodySerializer.class)```
   or ```@Request(accept=MyCustomResponseDeserializer.class)```.
-* Custom serialization from ```Map```, ```Multimap``` and custom types into query parameters. This functions
-  similarly to the Feign ```QueryMap``` feature, but with added control of customizing the serialization to
-  query parameters.
+* Custom serialization from ```Map```, ```Multimap``` and custom types into query parameters and header parameters. This functions
+  similarly to the Feign ```QueryMap``` feature, but with added control of customizing the serialization to query parameters and also with support for header parameters as well.
```

Comment:
```suggestion
  similarly to the Feign ```QueryMap``` and ```HeaderMap` features, but with added control of customizing the serialization.
```